### PR TITLE
Updates to new kafka-zookeeper image inlining healthcheck

### PR DESCRIPTION
**Pull request by adriancole**
note the succinct command for kafka-zookeeper

```bash
$ docker-compose ps
            Name                          Command                  State                                               Ports                                        
--------------------------------------------------------------------------------------------------------------------------------------------------------------------
all-views-creator              wait-for-it.sh -h pinot -p ...   Exit 0                                                                                              
all-views-generator            java -cp /app/resources:/a ...   Up (healthy)   8099/tcp                                                                             
attributes-bootstrapper        wait-for-it.sh -h hypertra ...   Exit 0                                                                                              
hypertrace-collector           /occollector_linux --confi ...   Up             0.0.0.0:14267->14267/tcp, 0.0.0.0:14268->14268/tcp, 55678/tcp, 0.0.0.0:9411->9411/tcp
hypertrace-federated-service   wait-for-it.sh -h pinot -p ...   Up (healthy)   0.0.0.0:2020->2020/tcp, 9001/tcp, 9002/tcp                                           
hypertrace-trace-enricher      java -cp /app/resources:/a ...   Up (healthy)   8099/tcp                                                                             
kafka-zookeeper                start-kafka-zookeeper            Up (healthy)   19092/tcp, 2181/tcp, 9092/tcp                                                        
mongo                          /root/run.sh mongod --bind ...   Up (healthy)   27017/tcp, 28017/tcp                                                                 
pinot                          ./bin/pinot-admin.sh Start ...   Up (healthy)   8096/tcp, 8097/tcp, 8098/tcp, 8099/tcp, 9000/tcp, 9514/tcp                           
raw-spans-grouper              java -cp /app/resources:/a ...   Up (healthy)   8099/tcp                                                                             
schema-registry                /usr/bin/docker-start.sh         Up (healthy)   8081/tcp                                                                             
span-normalizer                java -cp /app/resources:/a ...   Up (healthy)   8099/tcp                     
```

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -176,13 +176,8 @@ services:
   # Kafka is used for streaming functionality.
   # ZooKeeper is required by Kafka and Pinot
   kafka-zookeeper:
-    image: hypertrace/kafka-zookeeper:0.1.4
+    image: hypertrace/kafka-zookeeper
```
Comment (jcchavezs) [inline docker/docker-compose.yml:179]: We definitively need the CI running nightly on this.

Comment (adriancole) [reply docker/docker-compose.yml:179]: dude it is awesome you volunteer!

Comment (adriancole) [reply docker/docker-compose.yml:179]: seriously though I think we will change this to main tag once that is published on commit.. probably we need matrix latest and main to know impact

Comment (adriancole) [reply docker/docker-compose.yml:179]: ps not really sure why all but 2 are latest. but I will update those once touch them

**Review by JBAhire — APPROVED** [review]

**Pull request merged**